app-emulation/oricutron/oricutron-1.1_svn.recipe

```diff
@@ -18,7 +18,9 @@ PROVIDES="
 REQUIRES="
 	haiku
 	libsdl
+	libsdl >= 1.2.10
```

Comment:
This looks wrong anyway. It's probably not worth reusing DEPEND, just commenting it, or dropping it.